Assets/Scripts/Deck.cs

```diff
@@ -168,15 +166,20 @@ public void MakeCards()
         // Iterate through all of the card names that were just made
         for(int i = 0; i < cardNames.Count; i++)
         {
-            // Make the card and add it to the cards Deck
-            cards.Add(MakeCard(i));
+            if(i <= 12 || i >= 39)
+            {
+                cards.Add(MakeCardBlack(i));
+            } else
+            {
+                cards.Add(MakeCardRed(i));
+            }
```

Comment:
is there a reason you chose to go hard coding by value instead of checking the suit? What are the disadvantages to this model? what are the advanatages?